content/topics/intro-to-relational-databases/_index.md

```diff
@@ -1,15 +1,105 @@
 ---
-title: SQL
-ready: True
+title: Introduction to relational database
+ready: true
 ---
 
-## Tutorial
 
-Complete the DataCamp Intro and Intermediate SQL courses:
+# INTRODUCTION TO RELATIONAL DATABASE (SQL)
```

Comment:
Make sure you can run the hugo site.  Access this page and look at it with your eyes. Looks bad hey?

This will show up as the title of your page: "Introduction to relational database" it will be a big fancy title and take up lots of space.
Immediately afterwards this will show up with the same level of hugeness: "INTRODUCTION TO RELATIONAL DATABASE (SQL)"

This is bad because it looks bad and it's repetitive. Convince yourself that this is bad.

If you ever find yourself doing this
```
# Anything at all
```
in your markdown files: stop drop and roll

```
# THIS is not ok
## this is ok
### so is this
#### etc
```